### PR TITLE
Day3 Parameterised recurring variables

### PR DESCRIPTION
**Pull request by mariachiinajar**

### CODE REVIEW

File: day03-vpc/main.tf [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+resource "aws_route_table_association" "tf-private-rta" {
+  count = length(var.private_subnet_cidr)
+
+  subnet_id      = aws_subnet.tf-private[count.index].id
+  route_table_id = aws_route_table.tf-private-rt[count.index].id
+}
+
+output "subnet-count" {
+    value = length(var.public_subnet_cidr)
+}
```
Comment (geoffrey1330) [inline day03-vpc/main.tf:104]: could you please, add a newline at the end of file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: day03-vpc/main.tf (hunk 1 above)
Comment (RustyLobster) [reply day03-vpc/main.tf:104]: Added a new line EOF.

Comment (RustyLobster) [reply day03-vpc/main.tf:104]: Added a new line EOF

Comment (RustyLobster) [reply day03-vpc/main.tf:104]: Added a new line EOF.

**Pull request merged**